tests/spi/spi.go

```diff
@@ -238,4 +273,137 @@ var _ = framework.SPISuiteDescribe(Label("spi-suite"), func() {
 		})
 	})
 
+	for _, test := range ServiceAccountTests {
+		test := test
+
+		Describe("SVPI-406 - "+test.TestName, Ordered, func() {
+			BeforeAll(func() {
+				// Initialize the tests controllers
+				fw, err = framework.NewFramework(utils.GetGeneratedNamespace("spi-demos"))
+				Expect(err).NotTo(HaveOccurred())
+				namespace = fw.UserNamespace
+				Expect(namespace).NotTo(BeEmpty())
+			})
+
+			// Clean up after running these tests and before the next tests block: can't have multiple AccessTokens in Injected phase
+			AfterAll(func() {
+				if !CurrentSpecReport().Failed() {
+					Expect(fw.AsKubeAdmin.SPIController.DeleteAllBindingTokensInASpecificNamespace(namespace)).To(Succeed())
+					Expect(fw.AsKubeAdmin.SPIController.DeleteAllAccessTokensInASpecificNamespace(namespace)).To(Succeed())
+					Expect(fw.AsKubeAdmin.SPIController.DeleteAllAccessTokenDataInASpecificNamespace(namespace)).To(Succeed())
+					Expect(fw.AsKubeAdmin.CommonController.DeleteAllServiceAccountsInASpecificNamespace(namespace)).To(Succeed())
```

Comment:
are you sure about this? It will delete default serviceaccounts as well.

Reply:
I think it is not a problem since the namespace is created for each test